native/src/routes/search/components/SearchHeader.js

```diff
@@ -28,18 +28,24 @@ const BoxShadow: StyledComponent<{||}, ThemeType, *> = styled.View`
 type PropsType = {|
   theme: ThemeType,
   query: string,
-  closeSearchBar: () => void,
+  closeSearchBar: (query: string) => void,
   onSearchChanged: (query: string) => void,
   t: TFunction
 |}
 
-const SearchHeader = ({ theme, query, closeSearchBar, onSearchChanged, t }: PropsType) => (
-  <BoxShadow theme={theme}>
-    <HorizontalLeft theme={theme}>
-      <HeaderBackButton onPress={closeSearchBar} labelVisible={false} />
-      <ThemedSearchBar theme={theme} onChangeText={onSearchChanged} value={query} autofocus t={t} />
-    </HorizontalLeft>
-  </BoxShadow>
-)
+const SearchHeader = ({ theme, query, closeSearchBar, onSearchChanged, t }: PropsType) => {
+  const onClose = () => {
+    closeSearchBar(query)
+  }
```

Comment:
Not sure if this is necessary, the query is available in the parent component as well (and only passed as prop here).